app/assets/stylesheets/modules/style.scss

```diff
@@ -0,0 +1,88 @@
+@import "variables";
+
+body {
+  background-image: image-url("default.jpg");
+  background-repeat: no-repeat;
+  background-attachment:fixed;
+  background-size: cover;
+  text-align: center;
+  color: $white;
+  height: 100%;
+  width: 100%;
+  font-family: Lato;
+  position: absolute;
+}
+
+.container-wrapper {
+  background-color: rgba( $black, .4 );
+  text-align: center;
```

Comment:
Properties should be ordered background-color, height, min-height, text-align, width